packages/config/config/devices/0x0346/keypad.json

```diff
@@ -0,0 +1,387 @@
+// Ring Keypad
+// Param information taken from https://support.ring.com/hc/en-us/article_attachments/360051851012/Keypad_Zwave_UK.pdf
+{
+	"manufacturer": "Ring",
+	"manufacturerId": "0x0346",
+	"label": "Keypad",
+	"description": "Keypad",
+	"devices": [
+		{
+			"productType": "0x0101",
+			"productId": "0x0201"
+		},
+		{
+			"productType": "0x0101",
+			"productId": "0x0202"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Battery Report Interval",
+			"unit": "min",
```

Comment:
Units should be minutes, seconds, etc. not abbreviated. And please put them directly under valueSize.